Source/WebCore/platform/graphics/skia/PlatformDisplaySkia.cpp

```diff
@@ -115,6 +164,8 @@ class SkiaGLContext : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Ski
             m_skiaGLContext = WTFMove(glContext);
             m_skiaGrContext = WTFMove(grContext);
         }
+
+        m_sampleCount = initializeMSAASampleCount(m_skiaGrContext.get());
```

Comment:
Move this to the previous if, because MakeGL can fail, in which case m_skiaGrContext is nullptr.